quirks/websites-with-shared-credential-backends.json

```diff
@@ -288,6 +292,10 @@
         "kcls.overdrive.com",
         "kcls.org"
     ],
+    [
+        "klm.com",
+        "flyingblue.com"
```

Comment:
Up at the top of this file we already have this entry:

```
    [
        "airfrance.com",
        "flyingblue.com"
    ],
```

We should add `klm.com` there rather than creating a new entry.

Reply:
Fair point.